CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 TiUP Changelog
 
+## [1.1.0] 2020.08.28
+
+- Support config monitor components ([#712](https://github.com/pingcap/tiup/pull/712), [@lucklove](https://github.com/lucklove))
+- Set correct `deploy_dir` of monitoring agents when importing ansible deployed clusters ([#704](https://github.com/pingcap/tiup/pull/704), [@AstroProfundis](https://github.com/AstroProfundis))
+- Support debug scp with --natvie-ssh flag ([#689](https://github.com/pingcap/tiup/pull/689), [@lucklove](https://github.com/lucklove))
```

Comment:
This is a trivial update, I think we can omit it in the release note.

Reply:
OK